dataactvalidator/scripts/loadTas.py

```diff
@@ -136,8 +136,11 @@ def add_existing_id(data):
 
 
 def existing_id(row, existing):
-    """Check for a TASLookup which matches this `row` in the `existing` data.
-    :param existing: Dict[account_num, List[TASLookup]]"""
```

Comment:
No biggie, but heads up that `:param` is a [sphinx](https://en.wikipedia.org/wiki/Sphinx_(documentation_generator)). No need to keep it here, but that's where the format comes from.